vsintegration/src/vs/FsPkgs/FSharp.LanguageService/FSharp.LanguageService.Base/CodeWindowManager.cs

```diff
@@ -132,19 +124,15 @@ public class CodeWindowManager : IVsCodeWindowManager {
             }
         }
 #endif
-        /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager.DropDownHelper"]/*' />
         /// <summary>Return the optional TypeAndMemberDropdownBars object for the drop down combos</summary>
         internal TypeAndMemberDropdownBars DropDownHelper {
             get { return this.dropDownHelper; }
         }
-        /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager.CodeWindow"]/*' />
         /// <summary>Return the IVsCodeWindow associated with this code window manager.</summary>
         internal IVsCodeWindow CodeWindow {
             get { return this.codeWindow; }
         }
       
-        #region IVsCodeWindowManager methods
```

Comment:
this is a style pr i know, i'd like some code to discuss :smile: 

The c# codebase need a bit of love.
Now is a lot harder to read than needed. 

I dont like region, some do, but here are useful? dont think so, maybe for interface implementation (see below). We should try to slim the code when possible, not hide it

A common indentation style help, the c# is verbose but we can use some stuff like automatic properties when useful, etc

the FSharp.LanguageService.Base style is different than ProjectSystem.Base, let's use a coding style, and align the code, so is easier to read. 
Like [roslyn Coding style](https://github.com/dotnet/corefx/wiki/Coding-style) not this strict (who care of c#), but a vssettings/resharpstyle help alot.

I can revert the region for implementation of interface, like `#region IVsCodeWindowManager` np, but maybe is better use explicit interface implementation for com interface, self documenting.